pkg/daemon/daemon.go

```diff
@@ -931,14 +941,14 @@ func (dn *Daemon) drainNode() error {
 		if err != nil {
 			lastErr = err
 			glog.Infof("Cordon failed with: %v, retrying", err)
-			return false, nil
+			return false, err
```

Comment:
this will cause the backoff loop to exit early since err != nil. I believe we want to keep retrying within the backoff loop in case of error like originally intended